features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/extensions/mobileapp/mobileupload.jag

```diff
@@ -116,13 +116,16 @@
 					var information = JSON.parse(d);
 					mergeRecursive(meta,information);
 				}
+                response.contentType = "application/json";
 				print(meta);
 				log.debug("Successfully discovered");
```

Comment:
Though this is not your written code,noticed there are log.debug level entries without checking debug enabled.Ideally those entries has to be inside  if (log.isDebugEnabled()) { check.